src/Components/LoginSignup.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from "react";

```

Comment:
Great implementation of the authForm, when you add in routers, you will need to navigate to the form or posts.